src/lib/i18n/client.ts

```diff
@@ -33,3 +33,10 @@ export function useTranslation(ns: string = defaultNS) {
 }
 
 export const Trans = TransW
+
+export function getAvailableLanguages() {
+  const supportedLngs = i18next.options.supportedLngs
+  return typeof supportedLngs === "object"
+    ? supportedLngs.filter((i) => i !== "cimode")
```

Comment:
What is `cimode` used for?

Reply:
> If set to 'cimode', the output text will be the key.

https://www.i18next.com/overview/configuration-options#languages-namespaces-resources